en-US/snippets/2017/jan2017.lang

```diff
@@ -0,0 +1,43 @@
+## NOTE: reference bug 1319916, deadline Jan 6th, 2017
+## NOTE: Guidelines on localizing snippets: https://goo.gl/PluA97.  The tone is QUAC (Quick, Untamed, Alert, Clever). Do your best to make it local and culturally relevant.
+## Link URL: https://testpilot.firefox.com/; applicable to all snippets
+## MAX_LENGTH: 180; applicable to all snippets
```

Comment:
Uhm, why improvising? ;-)

You need to put this line before each snippet, otherwise the system won't know where to check.

If you want to put the other two comments at the beginning of the file, you need to use the format `## NOTE: your comment`, not the format of a standard comment.